### PR TITLE
Add live teasers

### PR DESCRIPTION
**Pull request by florianherrengt**
image

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (adgad) [inline .vscode/launch.json]: Is this file meant to be committed?

File: templates/partials/timestamp.html [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-			{{/if}}
-			<time data-o-component="o-date" class="o-date" data-o-date-format="time-ago-limit-4-hours" datetime="{{publishedDate}}">{{#dateformat "dddd, d mmmm, yyyy"}}{{publishedDate}}{{/dateformat}}</time>
-		</div>
-	{{/with}}
-{{/unless}}
+{{#with @nTeaserPresenter.timeObject}}
+	<div class="o-teaser__timestamp{{#if classModifier}} o-teaser__timestamp--{{classModifier}}{{/if}}">
+		{{#if status}}
+			<span class="o-teaser__timestamp-prefix">LIVE</span>
+		{{/if}}
```
Comment (adgad) [inline templates/partials/timestamp.html]: This should be {{status}}, and not hardcoded to LIVE

**Comment by florianherrengt** [comment]
Still not sure about the way to display `Live` between `display-concept` and `timestamp`

File: src/presenters/teaser-presenter.js [hunk 3]
```diff
@@ -110,6 +110,9 @@ const TeaserPresenter = class TeaserPresenter {
 			mods.push(hyphenatePascalCase(this.data.type));
 		}
 
+		if (this.data.type === 'LiveBlog') {
```
Comment (adgad) [inline src/presenters/teaser-presenter.js]: Should we be checking the status === 'inprogress' here as well? i.e. a closed liveblog shouldn't go red...

File: .vscode/launch.json (hunk 1 above)
Comment (florianherrengt) [reply .vscode/launch.json]: It is not… 🙃  https://github.com/Financial-Times/n-teaser/pull/146/commits/2197f2a124e3a760fa59ab10220b1c75fb3beda4

File: src/presenters/teaser-presenter.js (hunk 3 above)
Comment (florianherrengt) [reply src/presenters/teaser-presenter.js]: Fixed and I've also added a test https://github.com/Financial-Times/n-teaser/pull/146/commits/14ef5b39025762a38ebdef59558d68582348d934

File: templates/partials/timestamp.html (hunk 2 above)
Comment (florianherrengt) [reply templates/partials/timestamp.html]: https://github.com/Financial-Times/n-teaser/pull/146/commits/9eb1f69fde40685f010093001bf87720a0b69ae6

**Comment by florianherrengt** [comment]
@adgad Are we waiting for Monday to merge this?

**Comment by florianherrengt** [comment]
Only small teasers have red background. Top stories are waiting for design.

**Comment by adgad** [comment]
👍

**Pull request merged**